src/client/app/containers/view.mode.ts

```diff
@@ -45,6 +53,27 @@ export class ViewMode implements OnInit, OnDestroy {
         return environment.current.host.toLowerCase();
     }
 
+    get isGist() {
+        return this.snippet && this.snippet.gist;
+    }
+
+    get openInPlaygroundSupported() {
+        let host = environment.current.host.toUpperCase();
+        return host === HostType.EXCEL || host === HostType.WORD || host === HostType.POWERPOINT;
+    }
+
+    get openInHostString() {
+        return this.strings.openInHost.replace('{0}', environment.current.host);
+    }
+
+    get downloadAsHostFileString() {
+        return this.strings.downloadAsHostFile.replace('{0}', environment.current.host);
+    }
+
+    get urlString() {
+        return `URL: ${window.location.href}`;
```

Comment:
However, that does remind me:  @cshashwat , if it doesn't already, could you please make sure that the IMPORT screen accepts the view URL and extracts the appropriate data from it?  And also add it to the testing.md list.